zipline/src/commonMain/kotlin/app/cash/zipline/ZiplineApiMismatchException.kt

```diff
@@ -15,4 +15,18 @@
  */
 package app.cash.zipline
 
-class ZiplineApiMismatchException(override val message: String) : Exception(message)
+class ZiplineApiMismatchException(override val message: String) : Exception(message) {
+  internal companion object {
+    /**
+     * This magic value is detected by the bridge and replaced with an actionable list of available
+     * functions.
+     */
+    internal const val UNKNOWN_FUNCTION = "<unknown function>"
```

Comment:
Today I learned – even though the companion is `internal`, if these aren’t `internal` then they leak in the public API!